packages/airnode-examples/contracts/relay-security-schemes/Requester.sol

```diff
@@ -11,6 +11,7 @@ contract Requester is RrpRequesterV0 {
     mapping(bytes32 => address) public sponsorWalletAddress;
     mapping(bytes32 => uint256) public chainId;
     mapping(bytes32 => bytes32) public chainType;
+    mapping(bytes32 => bytes32) public theRelayedRequestId;
```

Comment:
This naming is inconsistent with others. What of we named all of theose `relayedXyz` (this one would be relayedRequestId)?

Reply:
Good idea, I've updated all of the names with the latest commit